augur/data/schema-export-v2.json

```
@@ -400,7 +400,7 @@
                             "description": "Sequence accession number",
                             "$comment": "terminal nodes only",
                             "type": "string",
-                            "pattern": "^[0-9A-Za-z-_]+$"
+                            "pattern": "^[0-9A-Za-z._-]+$"
```

With the `-` at the end, we would have to make sure to not append to this list but keep the `-` at the end. For example, if we ever want to allow `~` to pass, `^[0-9A-Za-z._-~]+$` would include the range `_` to `~` rather than 3 different characters. `_~-` would be correct in this case, but it's not as straightforward as appending.

This would "future-proof" for appending:

```suggestion
                            "pattern": "^[0-9A-Za-z-._]+$"
```

---

Ok, I've moved it back.

FWIW, my intent by moving the hyphen to the end was to make it clearer when _reading_ the code, not updating it. I agree the end position makes updating not "just append", but that didn't bother me because code is read many times more than written. (I guess I'm also just used to it, because it's a long-standing informal practice of mine to keep hyphens at the end of char classes.)